browsergym/experiments/src/browsergym/experiments/loop.py

```diff
@@ -576,12 +606,15 @@ def summary_info(self) -> dict:
     def get_screenshot(self, step: int, som=False) -> Image:
         key = (step, som)
         if self._screenshots.get(key, None) is None:
-            file_name = f"screenshot_{'som_' if som else ''}step_{step}.jpg"
-            self._screenshots[key] = Image.open(self.exp_dir / file_name)
+            file_name = f"screenshot_{'som_' if som else ''}step_{step}"
+            try:
+                self._screenshots[key] = Image.open(self.exp_dir / (file_name + ".png"))
+            except FileNotFoundError:
+                self._screenshots[key] = Image.open(self.exp_dir / (file_name + ".jpg"))
         return self._screenshots[key]
 
     def get_screenshots(self, som=False):
-        files = list(self.exp_dir.glob("screenshot_step_*.jpg"))
+        files = list(self.exp_dir.glob("screenshot_step_*"))
```

Comment:
```suggestion
        files = list(self.exp_dir.glob(f"screenshot_{"som_" if som else ''}step_*.png"))
```

Reply:
@recursix is this what you meant ?

Reply:
it is what I meant. Thanks :)

Not 100% it works in practice though.